libraries/importers/component-importer/src/main/java/org/eclipse/sw360/importer/ComponentCSVRecord.java

```diff
@@ -288,6 +286,34 @@ private boolean isSetClearingInformation() {
         return false;
     }
 
+    private boolean isSetEccInformation() {
```

Comment:
Yes, but then I'd have to rewrite this style everywhere in this file for consistency. I decided against it and made my changes blend in with the existing code.